dex/src/main/scala/com/wavesplatform/dex/api/http/routes/v0/MarketRoute.scala

```diff
@@ -36,9 +36,15 @@ import io.swagger.annotations._
 import javax.ws.rs.Path
 import scala.concurrent.{ExecutionContext, Future}
 
+case class Settings(settings: MatcherSettings, getActualTickSize: AssetPair => FutureResult[BigDecimal]) {
```

Comment:
1. There should be `actorResponseTimeout` and `orderRestrictions` instead of `settings`, because we want settings those related only to `MarketRoute` instead of global settings.
2. `getActualTickSize` is not settings. Settings is what you define in a configuration